tests/models/llama/test_modeling_llama.py

```diff
@@ -675,16 +685,25 @@ def test_model_13b_greedy_generation(self):
     @require_read_token
     def test_compile_static_cache(self):
         NUM_TOKENS_TO_GENERATE = 40
-        EXPECTED_TEXT_COMPLETION = [
-            "Simply put, the theory of relativity states that 1) the speed of light is constant, 2) the speed of light is the same for all observers, and 3) the laws of physics are the same for all observers.",
-            "My favorite all time favorite condiment is ketchup. I love it on everything. I love it on my eggs, my fries, my chicken, my burgers, my hot dogs, my sandwiches, my salads, my p",
-        ]
+        EXPECTED_TEXT_COMPLETION = {
+            7: [
+                "Simply put, the theory of relativity states that 1) the speed of light is constant, 2) the speed of light is the same for all observers, and 3) the laws of physics are the same for all observers.",
```

Comment:
This fails on A100

Reply:
on the A10 runners with the transformers-all-latest-gpu docker image it passed, might be an env issue?

Reply:
yes the ref is probably good for A10G, but not A100